src/tests/JIT/CodeGenBringUpTests/CodeGenBringUpTests_do.csproj

```diff
@@ -4,6 +4,7 @@
     <Optimize>False</Optimize>
```

Comment:
Shouldn't this be `<Optimize>True</Optimize>`?

Reply:
Yes, you're right.
```suggestion
    <Optimize>True</Optimize>
```